azure-quantum/setup.py

```diff
@@ -82,7 +82,8 @@
     extras_require={
         "qiskit": [
             "qiskit-ionq>=0.1.4",
-            "qiskit-terra>=0.19.1"
+            "qiskit-terra>=0.19.1",
+            "qiskit-qir"
```

Comment:
We'll need to specify here the version that includes the recent changes on the qiskit-qir package.